tests/integration/cli/env_test.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+from dusty.source import Repo
+
+from ...testcases import DustyIntegrationTestCase
+from ...fixtures import single_specs_fixture
+
+class TestEnvCLI(DustyIntegrationTestCase):
+    def setUp(self):
+        super(TestEnvCLI, self).setUp()
+        single_specs_fixture()
+        local_dir = Repo('github.com/app/a').managed_path
+        if not os.path.exists(local_dir):
+            os.makedirs(local_dir)
+        self.run_command('bundles activate bundle-a')
+
+    def tearDown(self):
+        self.run_command('stop')
+        self.run_command('env unset appa --all')
+        super(TestEnvCLI, self).tearDown()
+
+    def assertEnvInContainer(self, app_or_service, var, value):
+        env = self.exec_in_container(app_or_service, 'printenv {}'.format(var)).rstrip()
+        self.assertEqual(env, value)
+
+    def assertEnvNotInContainer(self, app_or_service, var):
+        env = self.exec_in_container(app_or_service, 'printenv {}'.format(var)).rstrip()
+        self.assertEqual(env, '')
+
+    def test_env_put_in_container(self):
+        self.run_command('env set appa pitcher pedro')
+        self.run_command('up --no-pull')
+        self.assertEnvInContainer('appa', 'pitcher', 'pedro')
+
+    def test_env_overrides_spec_environment(self):
+        self.run_command('up --no-pull')
+        self.assertEnvInContainer('appa', 'SPEC_VALUE', 'spec-specified-value')
+        self.run_command('env set appa SPEC_VALUE new-value')
+        self.run_command('up --no-pull')
+        self.assertEnvInContainer('appa', 'SPEC_VALUE', 'new-value')
```

Comment:
Can you rewrite this to only need to do one up, instead specifying two different env vars and only overwriting one?